### ISSUE TITLE
Length of nodename

### ISSUE THREAD

**Issue by chriskaliX** [aquasecurity/tracee#1307]
Hi! I've some doubts with the length of nodename, in tracee.bpf.c, it's  `char uts_name[TASK_COMM_LEN];` where the `TASK_COMM_LEN` is 16.
But in vmlinux.h it's 65(terminated by a zero), and it's 65 as well in my machine(Linux VM-4-13-ubuntu 5.4.0-89-generic). I found a [reference](https://manpages.ubuntu.com/manpages/xenial/man2/uname.2.html) which indicates that the length vary from release to release...
In the linux source code, here is the [definition](https://github.com/torvalds/linux/blob/5bfc75d92efd494db37f5c4c173d3639d4772966/include/uapi/linux/utsname.h), it seems that in recently kernel version, the length is 65...

I am still a little bit confuse about the length, but I think maybe the length of nodename should be 65? Thanks~

**Comment by yanivagman** [comment]
Hello @chriskaliX!
You are right that 16 chars is not enough to contain the uts name. But actually, it's full length in the kernel was 64 since the time we added it into tracee. The reason we added it back then was that we needed a way to have the container id, and as the container runtime sets it to the container id, 16 chars were enough. Today we extract the container id from the cgroup path name, so this becomes sort of redundant.
The thing is, that currently the event context size (context_t) is 104 bytes, so changing the uts name length to 64 will increase it to 152 which is almost 33% bigger per each event!
So here are the choices as I see it:
1. Keep the 16 chars size as is today
2. Remove the uts name from the context field

Removing the uts name from the context field has several options as well
1. Removing it from any event output for good.
2. Only showing the full uts name as a new argument in sched_process_exec/sched_process_fork
3. If we think that there is still a need to have it per each event, there are other things we can do, such as saving the uts name once at sched_process_fork and updating it whenever changed by the process. This will enable us to add it in userspace per each event, while saving the need to send it from the kernel per each event
